go.mod

```diff
@@ -3,11 +3,39 @@ module github.com/drone/drone-convert-starlark
 go 1.12
 
 require (
+	github.com/Masterminds/semver v1.5.0 // indirect
+	github.com/Masterminds/vcs v1.13.1 // indirect
+	github.com/Microsoft/go-winio v0.4.14 // indirect
+	github.com/armon/go-radix v1.0.0 // indirect
 	github.com/bazelbuild/bazel-gazelle v0.19.1
+	github.com/bmatcuk/doublestar v1.2.2 // indirect
+	github.com/boltdb/bolt v1.3.1 // indirect
+	github.com/cpuguy83/go-md2man/v2 v2.0.0 // indirect
+	github.com/docker/go-connections v0.4.0 // indirect
+	github.com/docker/go-units v0.4.0 // indirect
+	github.com/drone/drone-cli v1.2.0 // indirect
 	github.com/drone/drone-go v1.1.0
+	github.com/drone/drone-runtime v1.1.0 // indirect
+	github.com/drone/drone-yaml v1.2.3 // indirect
+	github.com/drone/envsubst v1.0.2 // indirect
+	github.com/fatih/color v1.9.0 // indirect
+	github.com/gogo/protobuf v1.3.1 // indirect
+	github.com/golang/dep v0.5.4 // indirect
+	github.com/google/go-jsonnet v0.14.0 // indirect
+	github.com/jmank88/nuts v0.4.0 // indirect
```

Comment:
something weird going on with dependencies here. Most of this stuff should not be required.